orders/src/main/java/com/orders/crudyorders/repos/CustomersRepository.java

```diff
@@ -0,0 +1,8 @@
+package com.orders.crudyorders.repos;
+
+import com.orders.crudyorders.model.Customer;
+import org.springframework.data.repository.CrudRepository;
+
+public interface CustomersRepository extends CrudRepository<Customer, Long>{
+    Customer findByCustname(String custname);
+}
```

Comment:
Line 7 - pay close attention to the method name here (which is correct): "findBy" + the field name to use for your search, with **only the first letter in uppercase**.